src/odemis/driver/andorcam2.py

```diff
@@ -3127,6 +3225,10 @@ def SetTriggerInvert(self, inverted):
         # We don't really have a hardware trigger, so no need to really invert it
         pass
 
+    def SetFastExtTrigger(self, fast):
```

Comment:
Well, that's the Andor2 SDK API. I'd rather keep the names of the arguments, even if the simulator doesn't care about it. It can be useful just to understand how the function should be used.